changelog/14946.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+**Secrets/auth plugin multiplexing**: manage multiple plugin configurations with a single plugin process
```

Comment:
I was reviewing the changelog for 1.12 features, and wasn't sure if the formatting of just:

**Plugin Multiplexing**:

would be more consistent with other new feature entries from older releases?